users/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.conf.urls import url
+from django.urls import path
 
-from users.views import current_user
+from . import views
 
 urlpatterns = [
-    url('current-user/', current_user)
+    path('current-user/', views.current_user, name='profile'), # replace with <int_user_id>
+    path('current-user/edit/', views.edit_current_user), # replace with <int:user_id>/update?
```

Comment:
The idea about current is that we can create a separate view that has access to much more information than the normal `id` (public) variant has. But I guess using `id` only across the whole site and adding a check for current works just as well 😊